### DISCUSSION TITLE
Disable Ligatures?

### DISCUSSION THREAD

**Discussion by ossfay** [geany/geany#2865] [Q&A] [answered]
There's a font I like to use for the Message Window area of Geany called Share Tech Mono. It's a free font I came across. I noticed a while ago that sometimes messages would look odd. for example a line would look like:
`13:43:52: File C:\MyProj\Make*le saved.`
where "Make*le" is supposed to be "Makefile"
I kept wondering why the letters 'f' and 'i' were turned into a dot. Then I learned about something called ligatures. Apparently the font I'm using doesn't have a character for that. Instead of using a different font I'd rather just turn off ligatures. Is that possible?

**Comment by elextr** [comment, marked as answer]
Geany has no control of ligatures, because it just writes the bytes `f` and `i` and its hidden in the GTK text widget and the font how its displayed.

Note the font says `fi` ligature fixed (see [here](https://fonts.google.com/specimen/Share+Tech+Mono)), are you sure that wherever you "came across" it has the latest?

Possibly changing your locale to `C` might fix it, or maybe GTK3 will fix it when the next windows release happens "soon", but really nothing should be ligating? (is that a word) in a monospace font, ligatures are inherently non-monospace.

**Reply by ossfay** [reply]
Ok. It was a font issue. I had done a search for "free monospace fonts" and I got the original file from some random free font website. It must have been old. I went to the link you provided to the Google font and downloaded that. I uninstalled the old one and installed the new one, and it seems fine now.
Still and odd issue though, because, as you mention, ligatures and monospace fonts don't seem to make much sense.
